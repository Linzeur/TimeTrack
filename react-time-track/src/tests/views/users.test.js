import React from "react";
import { render } from "@testing-library/react";
import AllUsers from "../../views/users";

test("Render all users", () => {
  const { asFragment } = render(<AllUsers />);
  expect(asFragment()).toMatchSnapshot();
});