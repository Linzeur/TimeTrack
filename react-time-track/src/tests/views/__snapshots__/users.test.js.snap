// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render all users 1`] = `
<DocumentFragment>
  <main>
    <section
      class="css-10oyw64-Section"
      role="list"
    >
      <button
        class="css-12phzq6-UserList"
      >
        New User
      </button>
    </section>
  </main>
</DocumentFragment>
`;
